platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -0,0 +1,125 @@
+import log from '../../log';
+import guid from '../../utils/guid';
+
+const EVENTS = {
+  MEASUREMENT_UPDATED: 'event@measurement_updated',
+  MEASUREMENT_ADDED: 'event@measurement_added',
+};
+
+const CONTEXTS = {
+  ALL: 'context@all',
+  VIEWER: 'context@viewer',
+  CORNERSTONE: 'context@cornerstone',
+};
```

Comment:
These would likely be set by the providers/subscribers themselves. Unfortunately a little brittle to set these here. Fine for stubbing/poc